src/lib/proof-system/zkprogram.ts

```diff
@@ -118,6 +123,13 @@ function createProgramState() {
   };
 }
 
+/**
+ * Initializes {@link Pickles} bindings, serializes the input proof and VK for use in Ocaml, then calls into {@link Pickles.verify} to verify the proof.
+ *
+ * @param proof Either a Proof instance or a JSON proof which gets converted into an {@link MlPair} of {@link FieldConst} arrays for use in the bindings.
```

Comment:
same here with the OCaml conversion types, I think ideally these things shouldn't be exposed to the user since they might just cause more confusion

Reply:
Removed the links here and skipped the reference to `MLFieldArray` - should be more generic now